.github/workflows/quality-report.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           
           brew install sonar-scanner
 
-          bundle exec fastlane test scheme:"OneLogin" \
+          bundle exec fastlane test scheme:"OneLoginStaging" \
             sonar_token:${{secrets.SONAR_TOKEN}} \
             workspace:${{github.workspace}} \
-            testplan:OneLogin
+            testplan:OneLoginStaging
```

Comment:
Am I right in thinking that we are only running the OneLoginStaging test plan in the pipeline, which includes unit tests plus the staging configuration tests? Should we not also run the build configuration tests?

Reply:
Or will build tests come separately, when we add the ui tests which run against the build version?

Reply:
I've reverted this to use the `OneLogin` Test Plan which contains only the unit tests as per our conversation.
The configuration tests for `staging` and `build`, as well as the UI/E2E tests are run on the merged action now